app/views/tutorials/configuring.pt.html.erb

```diff
@@ -0,0 +1,94 @@
+<h1>Tutorial</h1>
+<h2>Criando uma nova configuração</h2>
+
+<hr>
+
+<h3>Requisitos</h3>
+
+Para criar uma configuração, é necessário:
+<ul>
+  <li><%= link_to(t('signed_up'), new_user_registration_path) %> na plataforma</li>
+  <li>Estar <%= link_to(t('signed_in'), new_user_session_path) %></li>
+  <li>Já ter criado pelo menos um Grupo de Leitura</li>
+</ul>
+
+<hr>
+
+<h3>Introdução</h3>
+
+Para processar um repositório, o Mezuro utiliza configurações que você especifica no  <%= link_to(t('repository_creation'), tutorials_path('analyzing')) %>.
+
+<h4>Componentes de Configuração</h4>
+
+<p>Aqui você será apresentado brevemente a todos os componentes de uma configuração e, em seguida, será guiado através dos passos para criar uma nova configuração.</p>
```

Comment:
"...será guiado através dos passos..." is not a very good translation.
"...verá os passos para..." would be better.